### DISCUSSION TITLE
vx_images中的图片命名

### DISCUSSION THREAD

**Discussion by StarLeet** [tamlok/vnote#2051] [Q&A]
我该如何设置，才能让vx_images文件夹中的图片以时间戳的形式保存？

例如：202208138887...(年月日时分毫秒)
于我而言，这种命名方式看上去更加整齐便于后续的整理和阅读。

如果不建议这么做，能否告知一下这样命名的坏处？

🙏🙏感谢大家

**Comment by Fxiao1** [comment]
现在好像就是时间戳加下划线和一段后面的随机数命名的吧，比如“20200627001616034_7082.png”。而且
> 这种命名方式看上去更加整齐便于后续的整理和阅读。

需要自己整理图片后阅读图片文件名吗？vnote不就是帮我们干这个的么，都交给它吧，你自己整理之后不会通知到vnote，可能会导致有些图片不会随文档删除，或者引用不到图片了。
